src/Symfony/Component/Messenger/Transport/SenderInterface.php

```diff
@@ -20,8 +20,6 @@
 {
     /**
      * Sends the given envelope.
-     *
-     * @param Envelope $envelope
      */
-    public function send(Envelope $envelope);
```

Comment:
We don't have any use case in core for this, and the code is extensible enough so that people can write their own way of getter something back (ie a middleware + and extended sendInterface of their own, that's nothing standard/core anyway).
Better close the interface to ease maintaince from core team's pov.